examples/platform_channel/lib/main.dart

```diff
@@ -40,7 +40,7 @@ class _PlatformChannelState extends State<PlatformChannel> {
     eventChannel.receiveBroadcastStream().listen(_onEvent, onError: _onError);
   }
 
-  void _onEvent(String event) {
+  void _onEvent(Object event) {
```

Comment:
this one and the other _onEvent one too

Reply:
What's the invariant that says that this will always receive a String?  That is, why is `receiveBroadcastStream` typed to return `Stream<dynamic>` if this is always going to be a `String`?  Can I just change everything in `receiveBroadcastStream` from `dynamic` to `String`, or is there a more subtle invariant going on?

Reply:
Yeah I guess for this one Object is correct.

Reply:
`receiveBroadcastStream` is declared to return `Stream<dynamic>` because we in general receive JSON-like values (int/double/String/.../Lists of such/Maps of such). In this particular use case, we only send and receive Strings.